dev-site.yml

```diff
@@ -8,24 +8,41 @@ site:
 
 content:
   sources:
-    - url: .
-      branches: HEAD
+    - url: https://github.com/redhat-scholars/tekton-tutorial.git
+      # Stay on master to pull latest changes
+      branches: master
+      start_path: documentation
+    - url: https://github.com/kameshsampath/openshift-admins-devops.git
+      # PR branch
+      branches: tekton-pipelines
```

Comment:
This doesn't make much sense to me. Why would you pull the repo within the repo? Doesn't this also make local testing weird, especially without a tag?

Reply:
This is in total sense. This means the site you build is against the branch where I am having my PR at. No need for tags here.

Reply:
You can just checkout the PR submittter' PR and just build the site locally,  validate the content and comment. This way it also makes sure your local copy is sane with your changes and in sync with master. 

Ref: https://docs.github.com/en/enterprise/2.15/user/articles/checking-out-pull-requests-locally

Reply:
I have released the latest tekton tutorial and tagged the content to v0.14.3 which is OpenShift TP 1.1

Reply:
@thoraxe  @btannous ^^